### ISSUE TITLE
Final accuracy

### ISSUE THREAD

**Issue by hulianyuyy** [ycmin95/VAC_CSLR#1]
I want to make sure that you report 22.1 Dev WER and 23.0 Test WER, while 21.2 Dev WER and 22.3 Test WER of released pretrained model ? Thanks in advance for response!

**Comment by hulianyuyy** [comment]
I have witnessed the latest version of your paper where results are updated. This issue is to be closed.

**Comment by hulianyuyy** [comment]
However, i'd like to ask what it is to make you boost it from 22.1 to 21.2 compared to the last version of code?  Thanks in advance.

**Comment by ycmin95** [comment]
@hulianyuyy
We add ablations about the temporature and loss weight in supplementary, which can be found in Table 7 & 8 of our arXiv version.

**Comment by hulianyuyy** [comment]
May i also ask that the evaluation tool (python or scilte) will or won't affect the performance?

**Comment by ycmin95** [comment]
@hulianyuyy 
From my experience, the evaluation differences between these tools about WER are less than 0.1%.

**Comment by hulianyuyy** [comment]
Then how could i reproduce the results of iterative training as mentioned in tab.1 in the paper ?

**Comment by ycmin95** [comment]
@hulianyuyy 
You can follow their papers details and try to implement them, we didn't revised this part of codes.  Besides, there still a performance gap between my implementation and their results and we didn't pay much attention on this.

**Comment by hulianyuyy** [comment]
Many thanks for your reply of your work which will benefit the community a lot.
